metricbeat/module/system/_meta/config.yml

```diff
@@ -28,3 +23,9 @@
   period: 15m
   metricsets:
     - uptime
+
+# - module: system
```

Comment:
Nit: There is a space after # which we normally don't have except for comments.

Reply:
Ok, fixed.